### PR TITLE
Dashboard v2.15-incubation

### PR DESCRIPTION
**Pull request by andrewballantyne**
Release notes: https://github.com/opendatahub-io/odh-dashboard/releases/tag/v2.15.0-incubation

### CODE REVIEW

File: odh-dashboard/base/kustomization.yaml [hunk 1]
```diff
@@ -21,10 +21,11 @@ resources:
   - image-puller.clusterrolebinding.yaml
   - model-serving-role.yaml
   - model-serving-role-binding.yaml
+  - fetch-accelerators.rbac.yaml
 images:
 - name: odh-dashboard
   newName: quay.io/opendatahub/odh-dashboard
-  digest: sha256:4478d3830a30058fb87ed3e94dfdc615bf7517fddbd8490d6e2563263a0752fb
+  digest: sha256:a88ae7ba1b8c4ab2f0c3850b219ca1e031c73c1aaae79998358ae2a6850d7da5
```
Comment (andrewballantyne) [inline odh-dashboard/base/kustomization.yaml:28]: Out of date nightly -- need one more night to get the latest sha (this excludes the latest in `main`).

/hold

**Comment by andrewballantyne** [comment]
Or... it passes. Flakes are fun 🙂

**Comment by VaishnaviHire** [comment]
@andrewballantyne @lucferbux Is this for 1.10 release?

**Comment by lucferbux** [comment]
@VaishnaviHire Yes, that's right, this is targeted for next release.

**Review by lucferbux — APPROVED** [review]
/lgtm

**Comment by andrewballantyne** [comment]
> @andrewballantyne @lucferbux Is this for 1.10 release?

Did I not reference it properly on the issue you created? https://github.com/opendatahub-io/opendatahub-community/issues/104 

I sorta followed what Pipelines did. Was I intended to edit the description or check the box or something?

**Comment by andrewballantyne** [comment]
@anishasthana / @lavlas / @VaishnaviHire Can one of you approve this?

**Comment by VaishnaviHire** [comment]
> > @andrewballantyne @lucferbux Is this for 1.10 release?
> 
> @VaishnaviHire Did I not reference it properly on the issue you created? [opendatahub-io/opendatahub-community#104](https://github.com/opendatahub-io/opendatahub-community/issues/104)
> 
> I sorta followed what Pipelines did. Was I intended to edit the description or check the box or something?

As Operator will directly pull manifests from `odh-dashboard` repo for ODH 1.10 release, this should not be required.
Is there any difference in manifests as compared to the one in [v2.15.0-incubation](https://github.com/opendatahub-io/odh-dashboard/tarball/v2.15.0-incubation) tag?

**Pull request closed without merging**